cibuildwheel/__main__.py

```diff
@@ -402,6 +401,15 @@ def get_build_identifiers(
 def detect_warnings(*, options: Options, identifiers: Iterable[str]) -> list[str]:
     warnings = []
 
+    python_version_deprecation = ((3, 11), 3)
+    if sys.version_info[:2] < python_version_deprecation[0]:
+        python_version = ".".join(map(str, python_version_deprecation[0]))
+        msg = (
+            f"cibuildwheel {python_version_deprecation[1]}+ will require Python {python_version}+, "
+            "please upgrade your python version."
```

Comment:
```suggestion
            "please upgrade your Python version used to run cibuildwheel."
```

Reply:
looks good.

Reply:
I'm not completely sure the commit I just added is the best way to do that but that's the best I could think of.
Having the full matrix of targets just below the sentence "While cibuildwheel itself requires a recent enough python version to run, it can target the following versions to build wheels", I don't think there can be any confusion left.